Assets/Imports/Qualisys/QTM-Unity-Realtime-Streaming/Streaming/RTMarkerStream.cs

```diff
@@ -45,6 +47,12 @@ private void InitiateMarkers()
                 newMarker.transform.parent = markerRoot.transform;
                 newMarker.transform.localScale = Vector3.one * markerScale;
                 newMarker.SetActive(false);
+                // TDW added code from here to... 3/17/2022
```

Comment:
is this code from March 2022? Or today?

Reply:
I copied changes that I had made in the previous project back in March (changes that make QTM work with our Unity project) into the current (August) Ar_GroundPlane project.

Michael and I tested the code and it works like it used to.

Reply:
I'm more-or-less ok with it, as it seems pretty straightforward what it's doing (i.e. establishing each incoming marker as a colider and whatnot). 

...however, from a software dev perspective, it would be better to leave the Qualisys scripts as they are and have a different function that applies the 'collider' stuff. What you're doing here sorta violates both the 'single responsibility' and 'open-close principle' rules of S.O.L.I.D

... but again again - to emphasize - I'm ok with including this, but update the comment to clarify that you original wrote that code in March but copy-pasted it on the current date (will help with future debugging, if necessary)